### PR TITLE
Dw-102 install styleint

### PR DESCRIPTION
**Pull request by GusBaamonde**
library website: https://stylelint.io/

stylelint

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -44,13 +44,14 @@
...
     "styled-components": "^4.4.1",
+    "stylelint": "^13.0.0",
     "typescript": "^3.7.5",
     "url-parse": "^1.4.7"
   },
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
-    "verify": "yarn prettier-check && yarn eclint && yarn test:ci",
+    "verify": "yarn prettier-check && yarn eclint && yarn test:ci && yarn stylelint",
```
Comment (andresmoschini) [inline package.json]: 👏

**Comment by GusBaamonde** [comment]
test this please

**Comment by GusBaamonde** [comment]
test this please

**Pull request merged**